pelican/contents.py

```diff
@@ -383,21 +416,36 @@ def relative_dir(self):
 
 class Page(Content):
     mandatory_properties = ('title',)
+    allowed_status = ('published', 'hidden')
```

Comment:
Renaming this to `allowed_statuses` makes it clearer that each page can have multiple valid statuses (rather than just one).